index.js

```diff
@@ -185,7 +190,7 @@ class LogstashUDP extends Transport {
   }
 
   _buildLog(info) {
-    const meta = Object.assign({}, info, this.meta_defaults);
+    const meta = Object.assign(info, this.meta_defaults);
```

Comment:
`winston-transport` already does `Object.assign({}, info)` before passing it to the format, so it's already a fresh object and no need to dup it again.